drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -20,6 +20,9 @@ function DragAndDropBlock(runtime, element, configuration) {
     var M = 77;
     var QUESTION_MARK = 63;
 
+    // Event string size limit.
+    var MAX_LENGTH = 255;
+
```

Comment:
@Kelketek Can you move this block above the `// Keyboard accessibility` block please? The three vars below (`placementMode`, `$selectedItem`, `$focusedElement`) belong to the "Keyboard accessibility" section, and they are now separated from it.